### PR TITLE
Enhance Any Sequencer

### PR DESCRIPTION
**Pull request by jinchihe**
**Which issue is resolved by this Pull Request:** 
1. Corrent help infomation for  Any Sequencer
2. Enhance log format to be consistent with others, see more https://github.ibm.com/AILifecycle/tracker/issues/1774

**Description of your changes:**

**Environment tested:**

* Python Version (use `python --version`):
* Tekton Version (use `tkn version`):
* Kubernetes Version (use `kubectl version`):
* OS (e.g. from `/etc/os-release`):

**Checklist:**
- [ ] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention). 

- [ ] Do you want this pull request (PR) cherry-picked into the current release branch?

    [Learn more about cherry-picking updates into the release branch](https://github.com/kubeflow/pipelines/blob/master/RELEASE.md#cherry-picking-pull-requests-to-release-branch).

### CODE REVIEW

**Comment by jinchihe** [comment]
@pugangxa would you please help to take a look? Great thanks.

**Comment by jinchihe** [comment]
@fenglixa would you please take a look? Thanks

File: tekton-catalog/any-sequencer/cmd/root.go [hunk 1]
```diff
@@ -188,9 +187,9 @@ func Execute() {
 	// rootCmd represents the base command when called without any subcommands
 	var rootCmd = &cobra.Command{
 		Use:   "any-taskrun",
-		Short: "Watch taskrun and exit when any taskrun complete",
-		Long: `Watch taskrun and exit when any of below is true:
-			1: taskrun complete
+		Short: "Watch taskrun or run, and exit when any taskrun or run complete",
```
Comment (fenglixa) [inline tekton-catalog/any-sequencer/cmd/root.go:190]: Great that the usage was also updated

**Pull request merged**